src/Aspire.Hosting.Dapr/DaprDistributedApplicationLifecycleHook.cs

```diff
@@ -105,10 +104,6 @@ public async Task BeforeStartAsync(DistributedApplicationModel appModel, Cancell
                         fileName,
                         Command("run"),
                         daprAppPortArg(sidecarOptions?.AppPort),
-                        daprGrpcPortArg(sidecarOptions?.DaprGrpcPort?.ToString(CultureInfo.InvariantCulture)),
```

Comment:
This is simply unnecessary. These parameters will be added by code starting around line 190.